styles.css

```diff
@@ -0,0 +1,125 @@
+
+.mobile-header {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0.1px;
+  top: 0.1px;
+  background: #3c3a39;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-right: 48px;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  left: 0;
+  top: 92px;
+  color: #ff6b00;
+  background: #1c1a19;
+  padding: 120px 24px;
+}
+
+p {
+  position: static;
+  flex: 1;
+  width: 77px;
+  height: 24px;
+  left: 12px;
+  top: 10px;
+  margin: 20px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+}
+
+img {
+  padding-bottom: 0;
+  order: 5;
+  align-self: flex-end;
+}
+
+h1 {
+  position: static;
```

Comment:
- [ ] Please avoid modifying the default `position` value unless necessary, you can create the whole portfolio only with flex-box and grid. Modifying position property will make a hell working in the responsive design in the future.